src/sequencer/reply.rs

```diff
@@ -124,6 +122,9 @@ impl TryFrom<CodeReply> for Code {
 #[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
 #[serde(deny_unknown_fields)]
 pub struct Code {
+    // Unknown block hash results in empty abi represented as a JSON
```

Comment:
Is this something you've mentioned to Starkware? Sounds weird.

Reply:
I haven't yet, but I'll add it to the list of 504s and 500s that used to occur :+1: .